lua/pac3/core/shared/http.lua

```diff
@@ -70,6 +70,7 @@ function pac.FixUrl(url)
 		url = url:gsub([[^https?://dl.dropbox.com/]], [[https://www.dropbox.com/]])
 		url = url:gsub([[^https?://www.dropbox.com/s/(.+)%?dl%=[01]$]], [[https://dl.dropboxusercontent.com/s/%1]])
 		url = url:gsub([[^https?://www.dropbox.com/s/(.+)$]], [[https://dl.dropboxusercontent.com/s/%1]])
+		url = url:gsub([[^https?://www.dropbox.com/scl/(.+)$]], [[https://dl.dropboxusercontent.com/scl/%1]]) --Fix for new dropbox format.
 		return url
```

Comment:
This doesn't do anything, you mentioned in https://github.com/CapsAdmin/pac3/issues/1290 that you have to replace s with scl, but this effectively replaces scl with scl.

Reply:
Nevermind, I focused too much on the part after /, so this replaces dropbox.com with dl.dropboxusercontent.com